operator/config/webhook/manifests.yaml

```diff
@@ -27,45 +27,3 @@ webhooks:
     resources:
     - seldondeployments
   sideEffects: None
-- admissionReviewVersions:
```

Comment:
Is this remove expected?

Reply:
good spot. this should be kept

Reply:
Actually not clear as what exists presently seems to be duplicated sections

Reply:
Will revert and we can investigate in separate PR